src/morphodict/lexicon/migrations/0003_remove_unused_dictionarysource_fields.py

```diff
@@ -0,0 +1,35 @@
+from django.db import migrations
+
+
+class Migration(migrations.Migration):
```

Comment:
Since this is all coming in one PR, could you squash these linear migrations? It seems redundant to merge in code that adds columns to the database, and then immediately removes them.